core/commons/che-core-commons-inject/src/main/java/org/eclipse/che/inject/CheBootstrap.java

```diff
@@ -414,13 +414,13 @@ protected void bindProperties(String prefix, Map<String, String> properties) {
               } else if (skipUnresolved) {
                 buf.append(placeholder);
                 LOG.warn(
-                    "Placeholder {} cannot be resolved neither from environment variable nor from system property,"
+                    "Placeholder {} cannot be resolved neither from environment variable nor from system property, "
                         + "leaving as is.",
                     placeholderName);
               } else {
                 throw new ConfigurationException(
                     format(
-                        "Property %s is not found as system property or " + "environment variable.",
+                        "%s is not a system property or " + "an environment variable.",
```

Comment:
The string split here appears to be unnecessary.
```suggestion
                        "%s is not a system property or an environment variable.",
```

I would also nitpick `an` here, as dropping the indefinite article is fine here.